test/envs/python_bjoern/README

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
It's really a good idea to use spec here, since it's for benchmarking, how about putting these files to `$GOPATH/src/github.com/fission/fission/test/benchmark/assets`?